fenix/benchmark/build.gradle

```diff
@@ -0,0 +1,69 @@
+plugins {
+    id 'com.android.test'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'org.mozilla.fenix.benchmark'
+    compileSdk Config.compileSdkVersion
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+
+    defaultConfig {
+        minSdk 23
+        targetSdk Config.targetSdkVersion
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        // This benchmark buildType is used for benchmarking, and should function like your
+        // release build (for example, with minification on). It's signed with a debug key
+        // for easy local/CI testing.
+        benchmark {
+            debuggable = true
+            if (gradle.hasProperty("localProperties.autosignReleaseWithDebugKey")) {
+                signingConfig signingConfigs.debug
+            }
```

Comment:
Curious about 2 things here:
1. Does this module need to be debuggable for the profiling to work correctly? Just a little surprising given the emphasis on using release builds, but I didn't actually try the setup wizard
2. I think this `signingConfig` property will be added by the `releaseTemplate` closure used in the top-level [build file](https://github.com/mozilla-mobile/firefox-android/pull/842/files#diff-3eeb974dbea8ed2ac5d859292948457f089d0211ac697f866e4cf62c55576d4bR151). Is it necessary to have it in both places?

Reply:
1. Good question, on my Samsung A50 (android 11) the metrics can't be read/extracted when this is set to false. On Pixel 5 & 7 (android 13) it works regardless of this setting, so I set it true. The samples seem to do the [same](https://github.com/android/performance-samples/blob/main/MacrobenchmarkSample/macrobenchmark/build.gradle.kts#L40). 
2. This is the `benchmark/build.gradle`, the closure is defined in `app/build.gradle`, so that won't work right? Although i'm not sure if this condition is even required. This is something I'd like to get feedback on, based on how we sign apks on CI and how we would like to sign the benchmark apk as well.